packages/eas-cli/src/analytics/events.ts

```diff
@@ -15,6 +15,7 @@ export enum SubmissionEvent {
   SUBMIT_REQUEST_ATTEMPT = 'submit cli request attempt',
   SUBMIT_REQUEST_SUCCESS = 'submit cli request success',
   SUBMIT_REQUEST_FAIL = 'submit cli request fail',
+  CREDENTIALS_API_KEY_DOWNLOAD = 'submit cli credentials api key download',
```

Comment:
It looks like you are registering this even only on failure, so
```suggestion
  CREDENTIALS_API_KEY_DOWNLOAD_FAIL = 'submit cli credentials api key download fail',
```
and maybe a success event with a final number of retries would also be useful.

Also, maybe consider creating a separate group e.g. `CredentialsEvent`